docs/docs/policies.mdx

```diff
@@ -49,23 +49,23 @@ to the desired number of maximum predictions.
 
 In the case that two policies predict with equal confidence (for example, the Memoization
 and Rule Policies might both predict with confidence 1), the priority of the
-policies is considered. Rasa policies have default priorities that are set to ensure the
+policies is considered. Rasa Open Source policies have default priorities that are set to ensure the
 expected outcome in the case of a tie. They look like this, where higher numbers have higher priority:
 
-5. `RulePolicy`
+1. `TEDPolicy`
 
-3. `MemoizationPolicy` and `AugmentedMemoizationPolicy`
+2. `MemoizationPolicy` and `AugmentedMemoizationPolicy`
 
-1. `TEDPolicy`
+3. `RulePolicy`
 
 
 In general, it is not recommended to have more
 than one policy per priority level. If you have 2 policies with the same priority and they predict
 with the same confidence, the resulting action will be chosen randomly.
+(TODO: why we do it in number 2. then?)
```

Comment:
this is a placeholder: we say that it's a bad idea to have 2 policies with the same priority, but we give it as an example above

Reply:
oh, yes that's clearer!